fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java

```diff
@@ -89,12 +90,25 @@
      */
     private WhiteList whiteList = new WhiteList();
 
+    // the binding of sql_block_rule name, multiple are separated by ','
+    private String sqlBlockRules = null;
+
+    @Deprecated
```

Comment:
Why adding these deprecated fields back?

Reply:
The problem with rebase，I will change it